tensornetwork/backends/base_backend.py

```diff
@@ -285,3 +284,43 @@ def conj(self, tensor: Tensor) -> Tensor:
     """
     raise NotImplementedError("Backend '{}' has not implemented conj.".format(
         self.name))
+
+  def eigsh_lanczos(
+      self,
+      A: Callable,
+      initial_state: Optional[Tensor] = None,
+      ncv: Optional[int] = 200,
+      numeig: Optional[int] = 1,
+      tol: Optional[float] = 1E-8,
+      delta: Optional[float] = 1E-8,
+      ndiag: Optional[int] = 20,
+      reorthogonalize: Optional[bool] = False) -> Tuple[List, List]:
+    """
+    Lanczos method for finding the lowest eigenvector-eigenvalue pairs
+    of `A`. 
+    Args:
+      A: A (sparse) implementation of a linear operator. 
+      initial_state: An initial vector for the Lanczos algorithm. If `None`,
+        a random initial `Tensor` is created using the `torch.randn` method
+      ncv: The number of iterations (number of krylov vectors).
+      numeig: The nummber of eigenvector-eigenvalue pairs to be computed.
+        If `numeig > 1`, `reorthogonalize` has to be `True`.
+      tol: The desired precision of the eigenvalus. Currently we use 
+        `torch.norm(eigvalsnew[0:numeig] - eigvalsold[0:numeig]) < tol`
```

Comment:
Don't mention `torch` in the base backend.

Reply:
removed